### PR TITLE
Med

### PR DESCRIPTION
**Pull request by Outtacosmos-ai**

### CODE REVIEW

**Comment by Outtacosmos-ai** [comment]
My pull request

File: server/Models/Category.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+const mongoose = require("mongoose");
+
+const SchemaCategory = new mongoose.Schema({
+  productid: {
```
Comment (Samireddebbarhi) [inline server/Models/Category.js:4]: you should be product_id to make a clean code

File: server/Models/Customer.js [hunk 2]
```diff
@@ -1,2 +1,33 @@
...
+  },
+  password: {
+    type: String,
+    required: true,
+  },
+  dateofbirth: {
+    type: Date, // Changed to Date if it should represent a date
+    required: true,
+  },
+  premiumstatus: {
```
Comment (Samireddebbarhi) [inline server/Models/Customer.js:20]: the same

Comment (Samireddebbarhi) [inline server/Models/Customer.js:20]: the same point

File: server/Models/Customer.js [hunk 3]
```diff
@@ -1,2 +1,33 @@
...
+    type: Boolean,
+    required: true,
+  },
+  createdat: {
```
Comment (Samireddebbarhi) [inline server/Models/Customer.js:24]: createdAt

File: server/Models/Order.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
+const mongoose = require("mongoose");
+
+const SchemaOrder = new mongoose.Schema({
+  customerid: {
```
Comment (Samireddebbarhi) [inline server/Models/Order.js:4]: customer_id

File: server/Models/Order.js [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+    type: String,
+    required: true,
+  },
+  orderdate: {
```
Comment (Samireddebbarhi) [inline server/Models/Order.js:8]: same

File: server/Models/Order.js [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+    type: String,
+    required: true,
+  },
+  totalamount: {
```
Comment (Samireddebbarhi) [inline server/Models/Order.js:12]: ..

**Review by Samireddebbarhi — COMMENTED** [review]
repeat writting declaration for variable into clean code and be careful don't change any update to package.json to avaid any conflict while we'll merge it

**Pull request closed without merging**